pkg/ini.go

```diff
@@ -0,0 +1,228 @@
+package pkg
```

Comment:
`package pkg` doesn't indicate what the package does it should be something like iniparser